### ISSUE TITLE
Can there be a "leave as is" setting?

### ISSUE THREAD

**Issue by jmargolisvt** [atom/line-ending-selector#33]
I can't use Atom at work because files are a mix of LF and CRLF endings.  I don't want to set them all to any one type.  I just want them all left as they are.  Can we have a "leave line endings alone" setting?

**Comment by lee-dohm** [comment]
The Line Ending Selector doesn't do anything to the line endings unless you manually select a new line ending to convert to. So "leave line endings alone" is its default behavior, unless I'm misunderstanding what you're asking?

**Comment by jmargolisvt** [comment]
Maybe I'm misunderstanding something?  When I open a file at work (where others on Windows machines have edited files) and don't otherwise touch the file, I save it and git shows differences even thought I haven't touched the file except for open/save.  I thought this was a line ending problem?  I don't have the same issue with Sublime.

**Comment by jmargolisvt** [comment]
Figured this out.  Atom's Whitespace package comes with its default set to "remove trailing whitespace".  When I was hacking at home, no problem, just my code.  When I tried to use it with files that had trailing whitespace at work, git was seeing them as different any time I saved.

I prefer to have trailing whitespace removed, but using this as a default was pretty confusing for me, especially when the default behavior of Atom doesn't show display whitespace.  Git doesn't show the difference for whitespace either, so it was tough to trace this problem.
